.travis.yml

```diff
@@ -20,3 +20,7 @@ branches:
 cache:
   directories:
   - $HOME/.pub-cache
+
+env:
+  - USE_FLUTTER_SDK=false
```

Comment:
Remove the USE_FLUTTER_SDK=false
option from env

Reply:
maybe I'm missing what this does . if this is indeed running it with both options then that is fine.

Reply:
removed it to see if it would still run with both envs. If not, I will add it back.

Reply:
Looks like now it is only running with USE_FLUTTER_SDK=true. Adding it back.

Reply:
Actually on second thought, I think removing the environments altogether should work fine. As long as the tests requiring the flutter sdk are run last, we should not see the issue the env's were originally added to solve.